exporter/containerimage/exptypes/types.go

```diff
@@ -5,9 +5,14 @@ import (
 	specs "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
-const ExporterImageConfigKey = "containerimage.config"
-const ExporterInlineCache = "containerimage.inlinecache"
-const ExporterPlatformsKey = "refs.platforms"
+const (
+	ExporterConfigDigestKey      = "config.digest"
```

Comment:
This is technically not the same thing as the others. Still fine to have it in this pkg but maybe calling it `AnnotationConfigDigestKey` would be slightly clearer.